src/test/java/org/springframework/integration/aws/inbound/S3InboundChannelAdapterTests.java

```diff
@@ -191,7 +191,7 @@ public S3InboundFileSynchronizer s3InboundFileSynchronizer() {
 			synchronizer.setPreserveTimestamp(true);
 			synchronizer.setRemoteDirectory(S3_BUCKET);
 			synchronizer.setFilter(new S3RegexPatternFileListFilter(".*\\.test$"));
-			Expression expression = PARSER.parseExpression("#this.toUpperCase() + '.a'");
+			Expression expression = PARSER.parseExpression("(#this.contains('/') ? #this.substring(#this.lastIndexOf('/') + 1) : #this).toUpperCase() + '.a'");
```

Comment:
Hi Artem 

My pleasure.  Do you think the change will make it into the 1.0.1 release of spring-integration-aws? I'm considering "rolling-my-own" and deploying it for the time being to stick w/ my project schedule but I'd like to get back onboard with the official Spring release of sprint-integration-aws as soon as possible.